### ISSUE TITLE
Build libzim & libkiwix for Android with debug symbol

### ISSUE THREAD

**Issue by kelson42** [kiwix/java-libkiwix#121]
This is recommend by the PlayStore to be able to analyze the crash with the native code. It seems this is not the case (anymore?)

**Comment by MohitMaliFtechiz** [comment]
@kelson42 Yes, it is a valid problem. I checked our app bundle and found the `debug_symble` does not exist in our app bundle. I will place a fix for it shortly.

**Comment by kelson42** [comment]
@MohitMaliFtechiz Ok, please fix this ASAP so we are ready for the new release of java-libkiwix

**Comment by MohitMaliFtechiz** [comment]
@kelson42 I am on it.
